public/app/features/explore/Logs/Logs.tsx

```diff
@@ -146,6 +146,8 @@ const getDefaultVisualisationType = (): LogsVisualisationType => {
 };
 
 const PINNED_LOGS_LIMIT = 3;
```

Comment:
Also cc @grafana/explore-squad, can we increase the limit? The workflow of opening up datalinks from logs is something I've seen quite repetitive. Say you have 10 log lines, and then you open the trace of each on after another.

I'd say, either we increase the limit here, of we change the logic of pinned lines from datalinks, to overwrite the last pinned line that was caused by clicking on a datalink.

Reply:
Lets say 10, maybe? With https://github.com/grafana/grafana/pull/90213 we would get some insights how often users actually try and pin log lines, so we can always revert.

Reply:
Sounds great!

Reply:
Been updated to 10 in latest push